controllers/namespace/controller.go

```diff
@@ -19,15 +22,62 @@ type NamespaceReconciler struct {
 	Registry string
 }
 
+var excludedNamespaceList = []string{
+	// System namespaces
+	"istio-system",
+	"kube-node-lease",
+	"kube-public",
+	"kube-system",
+	"skiperator-system",
+	"config-management-system",
+	"config-management-monitoring",
+	"asm-system",
+	"anthos-identity-service",
+	"binauthz-system",
+	"cert-manager",
+	"gatekeeper-system",
+	"gke-connect",
+	"gke-system",
+	"gke-managed-metrics-server",
+	"resource-group-system",
+	// Bundles NetworkPolicies already
+	"kasten-io",
+	// TODO needs NetworkPolicies/Skiperator
+	"vault",
+	// TODO PoC, add NetworkPolicies after
+	"sysdig-agent",
+	"sysdig-admission-controller",
+	"instana-agent",
+	"kubecost",
+	"argocd",
+	"crossplane-system",
+	"upbound-system",
+	"instana-autotrace-webhook",
+	"fluentd", //POC
+}
+
+func (r *NamespaceReconciler) isExcludedNamespace(ctx context.Context, namespace string) bool {
+	configMapNamespacedName := types.NamespacedName{Namespace: "skiperator-system", Name: "namespace-exclusions"}
+
+	namespaceExclusionCMap, err := util.GetConfigMap(r.GetClient(), ctx, configMapNamespacedName)
+	if err != nil {
+		util.ErrDoPanic(err, "Something went wrong getting namespace-exclusion config map: %v")
+	}
+
+	nameSpacesToExclude := namespaceExclusionCMap.Data
+
+	exclusion, keyExists := nameSpacesToExclude[namespace]
+
+	return (keyExists && exclusion == "true") || slices.Contains(excludedNamespaceList, namespace)
```

Comment:
I'm not sure if this is the final version or if you're waiting for the discussion above on what to use as the source of truth.

I was expecting the `excludedNamespaceList` in code to be replaced by the deployed configmap. Is this for backwards compatibility? Do you mind explaining why we need both?

Reply:
i simply forgot removing the in-code list. update coming